tesseract_command_language/include/tesseract_command_language/joint_waypoint.h

```diff
@@ -287,10 +287,8 @@ class JointWaypoint
     if (lower_tolerance.size() == 0 || upper_tolerance.size() == 0)
       return false;
 
-    // Check if they are close to 0
-    Eigen::VectorXd range = upper_tolerance - lower_tolerance;
-    double sum = range.sum();
-    return (sum > 1e-5);
+    // Check if they are the same
+    return Eigen::IsApprox(lower_tolerance, upper_tolerance);
```

Comment:
Is this returning the opposite of what it did before? It seems like previously true meant that the values were different (i.e.  `sum` was larger than the tolerance) and now true means that they are the same

Reply:
Yep good catch it should be `!Eigen::IsApprox(lower_tolerance, upper_tolerance);`

Reply:
updated to leverage `almostEqualRelativeAndAbs`